docs/sql-migration-guide.md

```diff
@@ -224,6 +224,32 @@ license: |
 
   - Since Spark 3.0, when casting string value to integral types, including tinyint, smallint, int and bigint type, the leading and trailing white spaces(<= ACSII 32) will be trimmed before convert to integral values, e.g. `cast(' 1 ' as int)` results `1`. In Spark version 2.4 and earlier, the result will be `null`.
 
+  - Since Spark 3.0, we pad decimal numbers with trailing zeros to the scale of the column for Hive result, for example:
+    <table class="table">
+        <tr>
+          <th>
+            <b>Query</b>
+          </th>
+          <th>
+            <b>Spark 2.4 or Prior or Hive</b>
```

Comment:
why Hive here? isn't this default behavior of Hive? @wangyum

Reply:
Yes. I updated it.